pkgs/tools/virtualization/multipass/default.nix

```diff
@@ -60,6 +62,8 @@ stdenv.mkDerivation {
     ./lxd_socket_path.patch
     ./cmake_no_fetch.patch
     ./cmake_warning.patch
+    ./vcpkg_no_install.patch
+    ./test_unreachable_call.patch
```

Comment:
Maybe a little explanation on what are these patches doing and if they need to be removed at the next bump.
(I know that other patches in the list doesn't have that... sadly)

Reply:
Fixed in the latest commit -- I added comments for all of the patches.